controllers/imageclusterinstall_controller.go

```diff
@@ -562,6 +562,9 @@ func (r *ImageClusterInstallReconciler) writeClusterInfo(ctx context.Context, lo
 		log.Infof("created new infra ID %s", infraID)
 	}
 
+	// TODO: add proxy to SeedReconfiguration when it will be supported
+	ici.Spec.Proxy = r.updateNoProxy(ici.Spec.Proxy, cd.Spec.ClusterName, cd.Spec.ClusterName)
```

Comment:
Instead of updating the CR spec,let's make this update on the proxy settings we write to the seedReconfig (once it's available).
Changing the CR  spec is discouraged

Reply:
for sure, it was this way only cause we didn't have seed reconfiguration api yet